test/jdk/java/awt/EmbeddedFrame/EmbeddedFrameGrabTest/EmbeddedFrameGrabTest.java

```diff
@@ -103,7 +103,12 @@ private void init() throws Exception {
         robot.mouseRelease(InputEvent.BUTTON1_MASK);
         robot.delay(1000);
         if (!combo.isPopupVisible()) {
-            throw new RuntimeException("Combobox popup is not visible!");
+            robot.mousePress(InputEvent.BUTTON1_MASK);
```

Comment:
Does the description of the bug mean that the user should click twice to make the popup menu visible?